lib/linguist/languages.yml

```diff
@@ -3725,6 +3725,16 @@ Just:
   - ".just"
   ace_mode: text
   language_id: 128447695
+KCL:
+  type: programming
+  color: "#00ADD8"
+  tm_scope: source.kcl
+  filenames:
+  - ".k"
```

Comment:
This is for specific files which generally don't have an extension that matches that being added.

```suggestion
```

Reply:
thx. I'll do a bigger refactoring.